cra-server/functions/getAxiosInstance.js

```diff
@@ -6,9 +6,11 @@ const https = require("https");
 const fs = require("fs");
 const path = require("path")
 const getServerInfoFromEnv = require('./getServerInfoFromEnv');
+const getSecurityInfoFromEnv = require('./getSecurityInfoFromEnv');
 
-const cert = fs.readFileSync(path.join(__dirname, '../../') + "ssl/keys/server.cert");
-const key = fs.readFileSync(path.join(__dirname, '../../') + "ssl/keys/server.key");
+const security = getSecurityInfoFromEnv();
+const cert = fs.readFileSync(path.join(__dirname, '../../') + security.browser_cert);
```

Comment:
the docs need to clarify how the file can be specified in the given parameters
There should always be an option to specify an absolute path - for example we will likely need to MOUNT a filesystem from a kubernetes secret & it's much clearer/simple if that is specified in absolute terms as we use in egeria
(same comment for other parms)

The user would probably expect a relative path to be in relation to the directory where the server is run from - so if the above is needed to compensate for any directory switching done when when running node, then that's ok - perhaps just need to check for the user parm beginning with /

Reply:
Agreed - I will change